requirements.txt

```diff
@@ -4,5 +4,5 @@ Fabric==0.9.0
 nose==1.0.0
 rdflib==3.2.1
 argparse==1.1
--e git://github.com/mattbasta/fastchardet#egg=fastchardet
+fastchardet
```

Comment:
one or the other, as you prefer. For big changes, creating a new commit is easier to review, and then we squash before merging. For such a small change, amending is fine ;)